modules/social_features/social_group/modules/social_group_request/social_group_request.install

```diff
@@ -105,3 +107,18 @@ function social_group_request_update_13002(): void {
     ->load('flexible_group-group_manager');
   $group_manager->grantPermission('administer membership requests')->save();
 }
+
+/**
+ * Apply the new entity form display for group content type.
+ */
+function social_group_request_update_13003(): void {
+  $config_path = \Drupal::service('extension.list.module')->getPath('social_group_request') . '/config/static';
+  $source = new FileStorage($config_path);
+  /** @var \Drupal\Core\Config\StorageInterface $config_storage */
+  $config_storage = \Drupal::service('config.storage');
+
+  $data = $source->read('core.entity_form_display.group_content.group_content_type_7fcb76fdf61a9.default_13003');
+  if (is_array($data)) {
+    $config_storage->write('core.entity_form_display.group_content.group_content_type_7fcb76fdf61a9.default', $data);
+  }
```

Comment:
Can we switch this over to be consistent and use the update helper module instead?